release/release_after_kokoro.sh

```
@@ -0,0 +1,195 @@
+echo "#"
+echo "# Clear '$WORK_DIR'."
+echo "#"
+ask_proceed
+
+set -x
+rm -rf $WORK_DIR && mkdir $WORK_DIR
+set +x
+
+###############################################################################
+echo
+echo "#"
+echo "# Copy files built (signed) by Kokoro"
+echo "# into '$SIGNED_DIR'."
+echo "#"
+
+echo -n "Enter GCS bucket URL taken from the Kokoro page: "
+read GCS_URL
+GCS_BUCKET_URL=$( echo $GCS_URL | sed -e 's,.*\(signedjars/staging/prod/google-cloud-eclipse/ubuntu/release/[0-9]\+/[0-9-]\+\),\1,' )
+echo "GCS URL extracted: $GCS_BUCKET_URL"
+echo -n "Check if the URL is correct. "
+ask_proceed
+
+set -x
+mkdir $SIGNED_DIR && \
+    gsutil -m cp -R gs://$GCS_BUCKET_URL/gfile/signed/* $SIGNED_DIR
+set +x
+
+if [ 0 -eq $( ls -1 $SIGNED_DIR | wc -l ) ]; then
+    echo "'$SIGNED_DIR' is empty. Wrong GCS bucket URL provided?"
+    echo "Fix the problem and rerun the script. Halting."
+    exit 1
+fi
+
+###############################################################################
+echo
+echo "#"
+echo "# Verify if constants have been injected..."
+echo "#"
+ask_proceed
+
+set -x
+javap -private -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.appengine.login_0.1.0.*.jar \
+       -constants com.google.cloud.tools.eclipse.appengine.login.Constants
+javap -classpath $SIGNED_DIR/plugins/com.google.cloud.tools.eclipse.usagetracker_0.1.0.*.jar \
+       -constants com.google.cloud.tools.eclipse.usagetracker.Constants
+set +x
+
+echo "Check the output above. "
+ask_proceed
+
+###############################################################################
+echo
+echo "#"
+echo "# Run Eclipse from the command line to generate the following files:"
+echo "#"
+echo "#     $LOCAL_REPO/artifacts.xml"
+echo "#     $LOCAL_REPO/content.xml"
+echo "#     $LOCAL_REPO/features/*"
+echo "#     $LOCAL_REPO/plugins/*"
+echo "#"
+ask_proceed
+
+echo "The following steps require running an Eclipse binary."
+while true; do
```

Yeah, failing fast seems better. The version of the bundled Eclipse on my Laptop is too old (3.8), so I'd rather require explicitly setting ECLIPSE_HOME.